packages/dds/tree/src/simple-tree/api/schemaFactoryAlpha.ts

```diff
@@ -215,6 +215,10 @@ const schemaStaticsAlpha: SchemaStaticsAlpha = {
  *
  * Some private methods on `SchemaFactory` are intentionally duplicated here to avoid increasing their exposure to `protected`.
  * If we were to do so, they would be exposed on the public API surface of `SchemaFactory`.
+ *
+ * When building schema, when `options` is not provided, TCustomMetadata infers to unknown.
```

Comment:
Nit
```suggestion
 * When building schema, when `options` is not provided, `TCustomMetadata` infers to `unknown`.
```

Is "infers to" commonly used phrasing for this sort of thing? I would expect "is inferred as".